### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by AyushDhimann**
## Proposed change

Optimized the code a little as slicing is more efficient than replacing and made the path more concise and efficient rather than using os.path.join.

Your PR description goes here.

## Type of change

- [ ] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [x] Existing code/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good (`make pre-commit`)
- [ ] I've added tests to verify that the new code works and all tests pass locally (`make test`, `make tox`)
- [ ] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: scripts/l10n/generate_mo_files.py [hunk 1]
```diff
@@ -24,20 +24,11 @@ def run(self):
...
-                (
-                    sys.executable,
-                    os.path.join("scripts", "l10n", "msgfmt.py"),
-                    "-o",
-                    mo_file,
-                    po_file,
-                ),
-                check=True,
-            )
+            subprocess.run([sys.executable, "scripts/l10n/msgfmt.py", "-o", mo_file, po_file], check=True)
```
Comment (arkid15r) [inline scripts/l10n/generate_mo_files.py:31]: I prefer to keep this for compatibility with OSes that use other (not a "/") character as a path separator.

**Review by arkid15r — CHANGES_REQUESTED** [review]
Hi @AyushDhimann,
thanks for the PR!

Please see my comment below and run `make pre-commit` (listed in the PR checklist) to make sure the code is ready for review.

File: scripts/l10n/generate_mo_files.py (hunk 1 above)
Comment (AyushDhimann) [reply scripts/l10n/generate_mo_files.py:31]: Understood

**Comment by arkid15r** [comment]
Closing out in favor of https://github.com/dr-prodigy/python-holidays/pull/1033

**Pull request closed without merging**